### PR TITLE
Add DNS records to provisioned TF libvirt network

### PR DESCRIPTION
**Pull request by YuviGold**

### CODE REVIEW

File: terraform_files/baremetal/variables-libvirt.tf [hunk 1]
```diff
@@ -203,4 +203,10 @@ variable "provisioning_cidr6" {
   description = "IPv6 provisioning network from network pool for automated tests"
   type = string
   default = ""
-}
\ No newline at end of file
+}
+
+variable "libvirt_dns_records" {
+  type        = map(string)
```
Comment (osherdp) [inline terraform_files/baremetal/variables-libvirt.tf:209]: this is map between string to string?

**Review by osherdp — APPROVED** [review]

File: terraform_files/baremetal/variables-libvirt.tf (hunk 1 above)
Comment (YuviGold) [reply terraform_files/baremetal/variables-libvirt.tf:209]: Yes. It's host name points to the resolved name

**Review by osherdp — APPROVED** [review]

**Review by osherdp — APPROVED** [review]

**Pull request merged**